tools/lkl/fuzzers/binder/README.md

```diff
@@ -0,0 +1,84 @@
+# Android Binder fuzzer
+
+This folder contains implementation of libprotobuf-mutator-based Android Binder
+fuzzer.
+
+# Build instructions
+
+Android Binder fuzzer is based on `libprotobuf-mutator`
+[project](https://github.com/google/libprotobuf-mutator). Thus,
+to build the fuzzer you would need to checkout and build `libprotobuf-mutator`.
```

Comment:
Maybe add a reference to the `libprotobuf-mutator` version that was used, in case older/newer versions won't work.

Reply:
Good point, totally makes sense!

Reply:
Added reference to v1.4 version.